### ISSUE TITLE
Face Mask Detection using EfficientnetB3 

### ISSUE THREAD

**Issue by NEERAJAP2001** [geekquad/Image-Processing-OpenCV#194]
On an additional note, I'd like to add a Live Webcam Feature to detect masks using EfficientNetB3.

# Architecture : 

```
tf.keras.applications.EfficientNetB3(
    include_top=True, weights='imagenet', input_tensor=None,
    input_shape=None, pooling=None, classes=1000,
    classifier_activation='softmax', **kwargs
)

```

# Project Pipeline : 

![face mask detection- mtcnn](https://user-images.githubusercontent.com/65017645/114337677-305cdf00-9b6f-11eb-8e22-f53bc57e3b7a.png)

I'd like to work on this issue as part of GSSOC-21.

**Comment by geekquad** [comment]
Hey @NEERAJAP2001,
No pre-trained models are allowed. Moreover, this is not something we are looking for.
Thankyou.
